python/nx-cugraph/nx_cugraph/algorithms/bipartite/generators.py

```diff
@@ -48,13 +48,17 @@ def complete_bipartite_graph(n1, n2, create_using=None):
         nodes.extend(range(n2) if nodes2 is None else nodes2)
         if len(set(nodes)) != len(nodes):
             raise nx.NetworkXError("Inputs n1 and n2 must contain distinct nodes")
+    if nx.__version__[:3] <= "3.2":
+        name = f"complete_bipartite_graph({orig_n1}, {orig_n2})"
+    else:
+        name = f"complete_bipartite_graph({n1}, {n2})"
```

Comment:
I'm curious what `name` is used for and why it has to be different for newer versions of NX.  From looking at `from_coo()`, I think `name` is an attr that is applied to the new graph object, but I didn't notice how it's used from there. Would a brief comment be out-of-place here?

Reply:
Here is context for posterity: https://github.com/networkx/networkx/pull/7399